auto-update-pip-packages/requirements.txt

```diff
@@ -1,2 +1,2 @@
 OpenFisca-France[api] >= 18.0.0, < 19.0
```

Comment:
Why not >= 18.10 as in [update-api-cmd branch](https://github.com/openfisca/openfisca-france/tree/update-api-cmd) ?

Reply:
This is for the old API. The old API doesn't need 18.1.0